pages/blog.module.scss

```diff
@@ -0,0 +1,108 @@
+@import '../styles/variables';
+@import '../styles/mixins';
+
+.featuredOuter {
+  background: $color-dark;
+  color: $color-font;
+  margin-top: $header-height;
+
+  @include breakpoint(medium-up) {
+    margin: 0 -3.5rem;
+    padding: 7.25rem 3.5rem;
+  }
+
+  @include breakpoint(small) {
+    padding: 0 1rem 1.5rem;
+    text-align: start;
+  }
+}
+
+.secondaryOuter {
+  background: $color-black-700;
```

Comment:
I think this is the color that they wanted to make darker, right? I'm fine with doing so here and in the homepage as well.

Reply:
ah yeah you're right, i forgot to update here